src/library/scala/collection/mutable/HashMap.scala

```diff
@@ -507,6 +512,20 @@ class HashMap[K, V](initialCapacity: Int, loadFactor: Double)
     this
   }
 
+  override def mapValuesInPlace(f: (K, V) => V): this.type = {
```

Comment:
Won't this create a new bridge method? I don't think we can add this override in 2.13.x.

Reply:
You're right...